python/py-netcdf4/Portfile

```diff
@@ -8,7 +8,6 @@ name                py-netcdf4
 python.rootname     netCDF4
 version             1.6.5
 revision            0
-
```

Comment:
please no whitespace changes; an empty line here is perfectly fine and what we normally do